less/components/blocks.less

```diff
@@ -25,4 +50,30 @@
     &:hover {
         color: darken(@bordered-box-link-color, 15%);
     }
+}
+
+.flag-box {
```

Comment:
Couldn't we have used what we have for cards here? https://eclipsefdn-hugo-solstice-theme.netlify.app/components/cards/ all we need to do to align it properly is add a class that unsets left position, and sets right using whats in the left position

Reply:
Didn't know that existed, I'll using that instead. Thx.